pynest/nest/lib/hl_api_simulation.py

```diff
@@ -190,99 +190,134 @@ def SetKernelStatus(params):
+    use_wfr : bool
+        Whether to use waveform relaxation method
+    wfr_comm_interval : float
+        Desired waveform relaxation communication interval
+    wfr_tol : float
+        Convergence tolerance of waveform relaxation method
+    wfr_max_iterations : int
+        Maximal number of iterations used for waveform relaxation
+    wfr_interpolation_order : int
+        Interpolation order of polynomial used in wfr iterations
+
+    Synapses
+    max_num_syn_models : int, read only
+        Maximal number of synapse models supported
     sort_connections_by_source : bool
-        whether to sort connections by their source; increases construction
+        Whether to sort connections by their source; increases construction
         time of presynaptic data structures, decreases simulation time if the
         average number of outgoing connections per neuron is smaller than the
-        total number of threads (read/write)
+        total number of threads
     structural_plasticity_synapses : dict
         Defines all synapses which are plastic for the structural plasticity
         algorithm. Each entry in the dictionary is composed of a synapse model,
-        the pre synaptic element and the post synaptic element. (read/write)
+        the pre synaptic element and the post synaptic element
     structural_plasticity_update_interval : int
-        defines all synapses which are plastic for the structural plasticity
-        algorithm. Each entry in the dictionary is composed of a synapse model,
-        the pre synaptic element and the post synaptic element. (read/write)
+        Defines the time interval in ms at which the structural plasticity
+        manager will make changes in the structure of the network (creation
+        and deletion of plastic synapses)
+
+    Output
+    data_path : str
+        A path, where all data is written to (default is the current
+        directory)
+    data_prefix : str
+        A common prefix for all data files
+    overwrite_files : bool
+        Whether to overwrite existing data files
+    print_time : bool
+        Whether to print progress information during the simulation
+    network_size : int, read only
+        The number of nodes in the network
+    num_connections : int, read only, local only
+        The number of connections in the network
+    local_spike_counter : int, read only
+        Number of spikes fired by neurons on a given MPI rank since NEST was
+        started or the last ResetKernel. Only spikes from "normal" neurons
+        (neuron models with proxies) are counted, not spikes generated by
+        devices such as poisson_generator.
+
+    Miscelaneous
+    dict_miss_is_error : bool
+        Whether missed dictionary entries are treated as errors
+    keep_source_table : bool
+        Whether to keep source table after connection setup is complete
 
     Notes
     -----
     Some of the keywords in the kernel status dictionary are internally
```

Comment:
This text is at the beginning of the parameter list, no need to repeat it here. I am not quite sure if it fits best at the top or the bottom. @sarakonradi What do you think? Maybe at the bottom is early enough?

Reply:
I suggest keeping this section at the top, since the information is relevant to the user before looking at the list. However, I prefer the wording within the ``Notes`` section; it is more direct. @morales-gregorio could you perhaps move this text up?

Reply:
done!